scripts/metrics/metrics-agent.cjs

```
@@ -143,15 +143,29 @@ class ConfigurationLoader {
+    if (typeof tokenOrOptions === 'object') {
+      // New format: { token, owner, repo, cache }
+      token = tokenOrOptions.token;
+      this.owner = tokenOrOptions.owner;
+      this.repo = tokenOrOptions.repo;
+      cache = tokenOrOptions.cache || cache;
+    } else {
+      // Old format: (token, cache)
+      token = tokenOrOptions;
+    }
+
     this.token = token || process.env.GITHUB_TOKEN;
```

ESLint

[error] 'process' is not defined.

(no-undef)